CHANGES.yaml

```diff
@@ -1,3 +1,23 @@
+- 2.1.0:
+    - Added Juju 2.0-beta support.
+    - Updated all api calls to support Juju 1.x and 2.x-beta facades.
+    - Added the ability to create and switch between models in Juju 2.0.
+    - Updated terminology to match the Juju 2.0 terminology.
+    - >
+      Created user profile view which shows you your available models, bundles
+      and charms after logging into the charmstore.
+    - >
+      Added support for syntax highlighting in the charm details pages in the
+      charmbrowser when the charm author provides a Github Flavoured Markdown
```

Comment:
The official name is "GitHub Flavored Markdown", per https://guides.github.com/features/mastering-markdown/#GitHub-flavored-markdown